icmplib/sockets.py

```diff
@@ -28,13 +28,24 @@
 
 import socket, asyncio
 from struct import pack, unpack
-from time import time
+from time import time, perf_counter
 
 from .models import ICMPReply
 from .exceptions import *
 from .utils import PLATFORM_LINUX, PLATFORM_MACOS, PLATFORM_WINDOWS
 
 
+'''
+If OS Windows, then we use time.perf_counter(). Else - time.time()
+
+'''
+if PLATFORM_WINDOWS:
+    get_time = perf_counter
+else:
+    get_time = time
+
+    
+    
```

Comment:
I agree with @TerjeMjelde in https://github.com/ValentinBELYN/icmplib/issues/61#issuecomment-1672851143

> there's no need to differentiate between Linux and Windows here.